app/web_hub.go

```diff
@@ -452,8 +448,19 @@ func (h *Hub) Start() {
 						}
 					}
 				}
+				if msg.GetBroadcast().UserId != "" {
+					candidates := connIndex.ForUser(msg.GetBroadcast().UserId)
+					for _, webConn := range candidates {
+						broadcast(webConn)
+					}
+				} else {
+					candidates := connIndex.All()
+					for webConn := range candidates {
+						broadcast(webConn)
+					}
+				}
```

Comment:
Would it be safe to avoid `else` here?
```suggestion
					continue
				}
				candidates := connIndex.All()
				for webConn := range candidates {
					broadcast(webConn)
				}
```

Reply:
sure, that's great.